tests/unit/test_debugger.py

```diff
@@ -0,0 +1,60 @@
+import os
+from typing import Any
+import pytest
+
+from yls.debugger import Debugger
+from yls.hookspecs import ErrorMessage
+
+
+@pytest.mark.parametrize(
+    "expr, expected",
+    [
+        (
+            "pe.number_of_sections",
```

Comment:
This might create quite a bit of overhead. Generally this is another example of something where we try to guess which import is used based on an expression [here](https://github.com/avast/yls/blob/501e6fba25bb9d55c8527c1bfe77c2f004c503c0/yls/utils.py#L709).

Except in the link it's not an issue if we have a false positive. In this case, however, it's a matter of either identifying something as a cuckoo or not. If we have a false positive, we will show the wrong result to the user. So, to make sure, we would have to use something less prone to issues, and the only better thing we have are visitors. Except we can't do that either, because we no longer have access to the yaramod representation of an expression.

So back to the original topic. Realistically we are looking at a regex check that will in some cases make a mistake. Or maybe an overhaul of the original `YaramodExpressionExtractor` used to identify the expression, which would return not only the the expression to be evaluated, but also the preferred context to use.